script/download-node.js

```diff
@@ -110,7 +115,8 @@ var downloadNode = function(version, done) {
   }
 };
 
-downloadNode('v0.10.35', function(error) {
+var apmNodeVersion = process.env.APM_NODE_VERSION || '0.10.35';
```

Comment:
> Building native modules doesn't rely on the version of Node, both 0.10 and 0.12 are fine.

OK, that's good news. So we might as well simplify my changes and simply specify a single version to download? In other words, no reason to support it being overridden at via environment variables (or similar) at build time?